Resources/Storage/StorageFactory.py

```diff
@@ -89,7 +89,7 @@ def getStorages( self, storageName, pluginList = None ):
     if pluginList is None:
       pluginList = []
     if not self.vo:
-      return S_ERROR( 'Mandatory vo parameter is not defined' )
+      gLogger.warn( 'No VO information available' )
```

Comment:
It is used if it is defined ( for some possible VO specific parameters ). If not defined, nothing specific. For example, like that it allows usage by dirac_admin group which is not hooked up on any VO. We have to think more about VO-group-user relations still. However, for the moment this is more harm than help if the VO parameters is mandatory